lib/customer.rb

```diff
@@ -0,0 +1,14 @@
+class Customer
+	attr_reader :telephone_number
+
+	def telephone_number(input)
+		if input[0] == "0"
+			input[0] =''
+			input.insert(0,"+44")
+		elsif input[0] == "+" && input.length == 13
+			input
+		else
+			input.insert(0,"+")
+		end
+	end	
+end
```

Comment:
you have tab indenting here, please prefer 2 space indenting - also you have added lots of functionality not requested in the brief.  In general only add complexity like this when you find you really really really need it.